rudder-web/src/main/scala/com/normation/rudder/web/snippet/node/AcceptNode.scala

```diff
@@ -356,20 +352,23 @@ class AcceptNode {
         ),
         """,{ "sWidth": "60px" },{ "sWidth": "70px", "bSortable":false },{ "sWidth": "15px", "bSortable":false }"""
         ,true
-      ),
-      "accept" -> {if (seq.size > 0 ) { SHtml.ajaxButton("Accept into Rudder", {
-        () =>  showConfirmPopup(acceptTemplate, "confirmPopup")
-      }) % ("style", "width:170px")} else NodeSeq.Empty},
-      "refuse" -> {if (seq.size > 0 ) { SHtml.ajaxButton("Refuse", {
-        () => showConfirmPopup(refuseTemplate, "refusePopup")
-      }) } else NodeSeq.Empty},
-      "errors" -> (errors match {
+      )
+    , "accept" ->
+          SHtml.ajaxButton(
```

Comment:
For testimonial: having these test breaks the UI when no node are present, but does not change the behaviour when there was no node selected (i.e: it was just an UI inconsistancy when no nodes where available). So it's an enhancement to remove them.